Bio/PDB/Entity.py

```diff
@@ -89,6 +115,21 @@ def add(self, entity):
         self.child_list.append(entity)
         self.child_dict[entity_id] = entity
 
+    def reset_full_id(self):
```

Comment:
Do you think this should be a new public method, or denote it as private using a leading underscore? i.e. ``def _reset_full_id(self):``